arch/c4_container.md

```diff
@@ -0,0 +1,57 @@
+# Level 2: Radix System Container
+
+This page implement the second "C" in the [C4Model](https://c4model.com/#coreDiagrams): the **C**container Model.
+
+It recommended that you understand the [Context Model](c4_context.md) (first "C") before moving on to this page.
+
+Our Containers are implemented in [Docker](https://www.docker.com/) and thus are compliant with the [Open Container Initiative](https://www.opencontainers.org/) (OCI)
+
+# Single Point Of Failure (SPOF)
+
+One of the most important architectural benefits with OCI Containers is modularity. For example: it is relatively easy to switch out the Reverse Proxy Container (below) from NGINX to [HAProxy](http://www.haproxy.org/). Modularity enables alternate Core Node System compositions, which will minimise the risk of single point of failures in a Radix Network. The long-term plan is to have alternate implementations of all Containers including RadixCore.
+
+# Security
+
+Our belowed RadixCore is implemented in Java. Java is great and all that but it is not the best tool for all problems and purposes. To protect the RadixCore, we have introduced some specialised Containers (Reverse Proxy, Rate Limiter) in front of it.
```

Comment:
Personally, I would suggest removing `Java is great and all that but it is not the best tool for all problems and purposes.` to something like ` and instead expanding `To protect the RadixCore` to `To supplement and protect. ..`